extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -95,6 +97,7 @@ export const StateProperty = {
   CAN_SHOW_PREVIOUS_PAGE_HELP: 'canshowpreviouspagehelp',
   CAN_SHOW_SHARING_UIS: 'canshowsharinguis',
   CAN_SHOW_SYSTEM_LAYER_BUTTONS: 'canshowsystemlayerbuttons',
+  CAN_SHOW_SIDEBAR_BUTTON: 'canshowsidebarbutton',
```

Comment:
Looking over this again, would it be better if  detecting the presence of the sidebar was only done in the system layer code through a method that checks the parent element for a sidebar rather than using the store service at all?

Reply:
I think it's okay to use the store service for this